WebFeedGatewayPlugin.php

```diff
@@ -126,7 +128,8 @@ public function fetch($args, $request): bool
                     'latestDate' => $latestDate,
                     'feedUrl' => $request->getRequestUrl(),
                     'userGroups' => $userGroups,
-                    'includeIdentifiers' => $includeIdentifiers
+                    'includeIdentifiers' => $includeIdentifiers,
+                    'language' => locale_get_region(Locale::getLocale()) ? locale_get_primary_language(Locale::getLocale()) .  '-' . locale_get_region(Locale::getLocale()) : locale_get_primary_language(Locale::getLocale()),
```

Comment:
@jonasraoni, what about this solution?

Reply:
Another possible problem, that makes me think that it's better to make that method public is that perhaps some PHP locales are not very compatible with our ones.

I mean, we have `be@cyrillic`, but a call to `Locale::getScript('be@cyrillic')` won't bring what we would expect, while `be-Cyrl` will.